src/config/passport.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
+import {
+  Strategy as GoogleStrategy
+} from 'passport-google-oauth20';
+import env from './env-config';
+import Helpers from '../utils/helpers';
```

Comment:
This is not how we have been importing helpermethods in the codebase. Please import it correctly.